example.py

```diff
@@ -13,6 +15,12 @@ def send_metrics_via_proxy(proxy_client):
         42422.0, None, "localhost", None)
 
 
+def send_delta_counter_via_proxy(proxy_client):
+    proxy_client.send_delta_counter(
+        "python" + repr(sys.version_info[0]) + ".delta.proxy.counter",
```

Comment:
Just wonder, why `repr()` but not `str()`?

Reply:
I can change to `str()`, if you believe that's better practice.

Reply:
In case like this, `str()` is common practice, unless you want to see object specific representation.
In most cases both `str()` and `repr()` produce identical results, but there might be exceptions like
```
>>> import datetime
>>> str(datetime.datetime.now())
'2018-08-22 19:01:36.898581'
>>> repr(datetime.datetime.now())
'datetime.datetime(2018, 8, 22, 19, 1, 48, 48232)'
```